mypy/stubgen.py

```diff
@@ -636,23 +643,26 @@ def main() -> None:
                                      pyversion=options.pyversion,
                                      no_import=options.no_import,
                                      search_path=options.search_path,
-                                     interpreter=options.interpreter)
+                                     interpreter=options.interpreter,
+                                     include_private=options.include_private)
         except Exception as e:
             if not options.ignore_errors:
                 raise e
             else:
                 print("Stub generation failed for", module, file=sys.stderr)
 
 
-def parse_options() -> Options:
-    args = sys.argv[1:]
+def parse_options(args: Optional[List[str]] = None) -> Options:
```

Comment:
Correct.  Would you prefer that I make args non-optional and pass `sys.argv` from `main`?